docs/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.rst

```diff
@@ -1,7 +1,237 @@
 :orphan:
 
+OmniSciOnNative execution
+=========================
+
+OmniSciDB is an open-source SQL-based relational database designed for the
+massive parallelism of modern CPU and GPU hardware. Its execution engine
+is built on LLVM JIT compiler.
+
+OmniSciDB can be embedded into an application as a dynamic library that
+provides both C++ and Python APIs. A specialized in-memory storage layer
+provides an efficient way to import data in Arrow table format.
+
+`OmniSciOnNative` execution uses OmniSciDB for both as a storage format and for
+actual data transformation.
+
+Relational engine limitations
+-----------------------------
+
+Using a relational database engine implies a set of restrictions on
+operations we can execute on a dataframe.
+
+1. We cannot handle frames that use data types not supported by OmniSciDB.
+Currently, we allow only integer, float, string, and categorical data types.
+
+2. Column data should be homogeneous.
+
+3. Can only support operations that map to relational algebra. This means
+most operations are supported over a single axis (axis=0) only. Non-relational
+operations like transposition and pivot are not supported.
```

Comment:
something's wrong with this list, see rendered:
![image](https://user-images.githubusercontent.com/45396231/144870567-60f1e7c3-07b5-4c5d-8701-ef6c5ac0d343.png)

Reply:
fixed, now all of the list items have proper indentation:
![image](https://user-images.githubusercontent.com/62142979/145211421-d8c7e239-cdd4-4af4-b5dd-ec3c88a505b5.png)